src/ui/server/controllers/insurance/business/map-and-save/company-different-trading-address/index.ts

```diff
@@ -15,11 +15,17 @@ const companyDifferentTradingAddress = async (formBody: RequestBody, application
   try {
     if (hasFormData(formBody)) {
       const dataToSave = mapSubmittedData(formBody);
+      /**
+       * maps errors for companyDifferentAddress
+       * Adds errors for fullAddress to validationErrors object
+       * As field is fullAddress in database
+       */
```

Comment:
I think this comment could just be 1 line or removed completely, as `mapErrors` has documentation

I just thought - should we have the field ID in the UI called `FULL_ADDRESS`? Would avoid this mapping/reduce complexity

Reply:
changed to FULL_ADDRESS